internal/testutil/mock_flags.go

```diff
@@ -0,0 +1,77 @@
+package testutil
+
+import "strings"
+
+type MockFlagsWithComma struct {
+	Ary     []string
```

Comment:
```suggestion
	Arg     []string
```

Reply:
or `Args`?

Reply:
Actually it means 'Array', not 'arg' or 'args', just for illustrate slice type field